notebooks/README.md

```diff
@@ -0,0 +1,25 @@
+# Spark and Jupyter Notebooks on Dataproc
+
+Google Cloud Dataproc allows you use Jupyter Notebooks on your cluster by enabling the Jupyter component and component gateway. 
+
+This folder contains example notebooks for using Spark with BigQuery Storage and Google Cloud Storage, and other common use cases.
```

Comment:
Please also add links for either the connectors or those services' docs directly.

Reply:
I have updated this